multiraft/multiraft_test.go

```diff
@@ -392,6 +392,70 @@ func TestMembershipChange(t *testing.T) {
 		}*/
 }
 
+func TestRemoveLeader(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	stopper := stop.NewStopper()
+	cluster := newTestCluster(nil, 6, stopper, t)
+	defer stopper.Stop()
+
+	// Consume and apply the membership change events.
+	for i := 0; i < 4; i++ {
```

Comment:
Copy/paste error from the previous test. It should be six (although as it turns out only the first three are really important).